cyb_oko/settings.py

```diff
@@ -36,6 +36,7 @@
     'voucher',
     'z',
     'members',
+    'intern',
```

Comment:
Since the information in it is more about what kind of an intern a user is then what kind of accesslevels it should have. But yeah, it's a bad branch name